plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -40,30 +40,19 @@
 #include <pcre.h>
 #endif
 
-#define LOG_PREFIX "regex_revalidate"
+#define PLUGIN_TAG "regex_revalidate"
+#define DEFAULT_CONFIG_NAME "regex_revalidate.config"
 #define CONFIG_TMOUT 60000
 #define FREE_TMOUT 300000
 #define OVECTOR_SIZE 30
 #define LOG_ROLL_INTERVAL 86400
 #define LOG_ROLL_OFFSET 0
 
-static inline void *
-ts_malloc(size_t s)
-{
-  return TSmalloc(s);
-}
-
-static inline void
-ts_free(void *s)
-{
-  return TSfree(s);
-}
-
 typedef struct invalidate_t {
   const char *regex_text;
   pcre *regex;
   pcre_extra *regex_extra;
-  time_t epoch;
+  time_t refresh;
```

Comment:
as "epoch" is merely a time format... I think the purpose for the time is more intuitive.